docs/docsite/rst/reference_appendices/interpreter_discovery.rst

```diff
@@ -15,37 +15,19 @@ To control the discovery behavior:
 
 Use one of the following values:
 
+auto (default) :
+  Searches the list (i.e. ``interpreter_python_fallback``) of common Python
+  interpreter paths and uses the first one found. Also issues a warning that
+  future installation of another Python interpreter could alter the one chosen.
+
 auto_legacy :
-  Detects the target OS platform, distribution, and version, then consults a
-  table listing the correct Python interpreter and path for each
-  platform/distribution/version. If an entry is found, and ``/usr/bin/python`` is absent, uses the discovered interpreter (and path). If an entry
-  is found, and ``/usr/bin/python`` is present, uses ``/usr/bin/python``
-  and issues a warning.
-  This exception provides temporary compatibility with previous versions of
-  Ansible that always defaulted to ``/usr/bin/python``, so if you have
-  installed Python and other dependencies at ``/usr/bin/python`` on some hosts,
-  Ansible will find and use them with this setting.
-  If no entry is found, or the listed Python is not present on the
-  target host, searches a list of common Python interpreter
-  paths and uses the first one found; also issues a warning that future
-  installation of another Python interpreter could alter the one chosen.
-
-auto : (default in 2.12)
-  Detects the target OS platform, distribution, and version, then consults a
-  table listing the correct Python interpreter and path for each
-  platform/distribution/version. If an entry is found, uses the discovered
-  interpreter.
-  If no entry is found, or the listed Python is not present on the
-  target host, searches a list of common Python interpreter
-  paths and uses the first one found; also issues a warning that future
-  installation of another Python interpreter could alter the one chosen.
-
-auto_legacy_silent
-  Same as ``auto_legacy``, but does not issue warnings.
-
-auto_silent
+  Deprecated alias for ``auto``.
+
+auto_silent :
   Same as ``auto``, but does not issue warnings.
 
-You can still set ``ansible_python_interpreter`` to a specific path at any
-variable level (for example, in host_vars, in vars files, in playbooks, and so on).
-Setting a specific path completely disables automatic interpreter discovery; Ansible always uses the path specified.
+auto_legacy_silent :
+  Deprecated alias for ``auto_silent``.
+
+``/path/to/python`` :
+  Use the specified path to Python.
```

Comment:
Rather than adding a placeholder path here, this could be described before the list of values with special meaning.

```suggestion
```